CRM/Activity/BAO/Activity.php

```diff
@@ -935,219 +888,6 @@ public static function activityComponents() {
   }
 
   /**
-   * Get the activity Count.
-   *
-   * @param array $input
-   *   Array of parameters.
-   *    Keys include
-   *    - contact_id  int            contact_id whose activities we want to retrieve
-   *    - admin       boolean        if contact is admin
-   *    - caseId      int            case ID
-   *    - context     string         page on which selector is build
-   *    - activity_type_id int|string the activity types we want to restrict by
-   *
-   * @return int
-   *   count of activities
-   */
-  public static function &getActivitiesCount($input) {
```

Comment:
The removal of this function is causing a test failure. I kind of like keeping it as a wrapper to the other - I think passing the $isCount param (or any other param to change the output)  is a bit yuck, but  I think it makes sense. However calling it via a wrapper function seems clear